### PR TITLE
Add com.github.rigellute.spotify-tui

### PR DESCRIPTION
**Pull request by Odilhao**
This package was created with the intention of having another alternative to install the [spotify-tui](https://github.com/Rigellute/spotify-tui) client. 

The author is ok with the package being created by [another](https://github.com/Rigellute/spotify-tui/issues/160) person

Please let me know if any changes needs to be made on this repo for the pull request to be accepted.

### CODE REVIEW

File: com.github.rigellute.spotify-tui.desktop [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Type=Application
+Name=Spotify TUI
+GenericName=Online music streaming service
+Comment=Access all of your favorite music from terminal
+Icon=com.spotifytui.Client
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.desktop]: Wrong name.

File: com.github.rigellute.spotify-tui.desktop [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+TryExec=spt
+Exec=spt %U
+Terminal=true
+MimeType=x-scheme-handler/spotify;
+Categories=Audio;Music;AudioVideo;
+Keywords=Music;Player;Streaming;Online;
+StartupWMClass=Spotify
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.desktop]: Incorrect.

File: com.github.rigellute.spotify-tui.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+command: spt
+id: com.github.rigellute.spotify-tui
+separate-locales: false
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.yaml]: Why not?

File: com.github.rigellute.spotify-tui.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+finish-args:
+  - "--share=network"
+  - "--filesystem=home"
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.yaml]: What does an streaming client need full access to home for?

File: com.github.rigellute.spotify-tui.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    - type: file
+      path: com.github.rigellute.spotify-tui.desktop
+    buildsystem: simple
+    build-commands: 
+      - "tar xvfz spotify-tui-linux.tar.gz"
+      - "install -D spt /app/bin/spt"
+      - "install -Dm644 com.github.rigellute.spotify-tui.svg /app/share/icons/hicolor/scalable/apps/com.github.rigellute.spotify-tui.svg"
+      - "install -Dm644 com.github.rigellute.spotify-tui-symbolic.svg /app/share/icons/hicolor/symbolic/apps/com.github.rigellute.spotify-tui-symbolic.svg"
+      - "install -Dm644 com.github.rigellute.spotify-tui.desktop /app/share/applications/com.github.rigellute.spotify-tui.desktop"
+runtime: org.freedesktop.Platform
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.yaml]: Please put these at the top, see other manifests for general sorting.

File: com.github.rigellute.spotify-tui.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    sources:
+    - sha256: 71e8e962bc9175003e01de1da4aafeab66d6dd7f40511ce32d734646a98b3e04
+      type: file
+      url: https://github.com/Rigellute/spotify-tui/releases/download/v0.13.0/spotify-tui-linux.tar.gz
+    - type: file
+      path: com.github.rigellute.spotify-tui.svg
+    - type: file
+      path: com.github.rigellute.spotify-tui-symbolic.svg
+    - type: file
+      path: com.github.rigellute.spotify-tui.desktop
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.yaml]: Can you get these upstreamed?

File: com.github.rigellute.spotify-tui.yaml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+separate-locales: false
+finish-args:
+  - "--share=network"
+  - "--filesystem=home"
+modules:
+  - name: spt
+    sources:
+    - sha256: 71e8e962bc9175003e01de1da4aafeab66d6dd7f40511ce32d734646a98b3e04
+      type: file
+      url: https://github.com/Rigellute/spotify-tui/releases/download/v0.13.0/spotify-tui-linux.tar.gz
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.yaml]: You should try building from source.

File: com.github.rigellute.spotify-tui.yaml (hunk 4 above)
Comment (Odilhao) [reply com.github.rigellute.spotify-tui.yaml]: The configuration file for the client is created at ${HOME}/.config/spotify-tui/client.yml.

File: com.github.rigellute.spotify-tui.yaml [hunk 8]
```diff
@@ -0,0 +1,28 @@
+command: spt
+id: com.github.rigellute.spotify-tui
+separate-locales: false
+finish-args:
```
Comment (TingPing) [inline com.github.rigellute.spotify-tui.yaml]: This has no audio access...?

File: com.github.rigellute.spotify-tui.yaml (hunk 4 above)
Comment (TingPing) [reply com.github.rigellute.spotify-tui.yaml]: Then that is a bug that should be fixed upstream. It should respect `$XDG_CONFIG_HOME` being set.

File: com.github.rigellute.spotify-tui.yaml (hunk 8 above)
Comment (bilelmoussaoui) [reply com.github.rigellute.spotify-tui.yaml]: Because it can't play audio. It basically let you switch musing from terminal

Comment (Odilhao) [reply com.github.rigellute.spotify-tui.yaml]: This is just a cli frontend for spotify, you still need some daemon/program to play the music, like Spotify or spotifyd.

File: com.github.rigellute.spotify-tui.yaml (hunk 6 above)
Comment (Odilhao) [reply com.github.rigellute.spotify-tui.yaml]: I'll be removing these icons since I couldn't find any with correct license.

Comment (TingPing) [reply com.github.rigellute.spotify-tui.yaml]: Alright, well it needs some icon. Probably should work with upstream on that.

**Comment by TingPing** [comment]
Missing an appdata file https://www.freedesktop.org/software/appstream/docs/.

File: com.github.rigellute.spotify-tui.yaml (hunk 4 above)
Comment (Odilhao) [reply com.github.rigellute.spotify-tui.yaml]: I'll see if I can make it work with the $XDG_CONFIG_HOME

File: com.github.rigellute.spotify-tui.yaml (hunk 7 above)
Comment (Odilhao) [reply com.github.rigellute.spotify-tui.yaml]: I'll try to reproduce the Rust build from source.

**Pull request closed without merging**